nautobot/core/settings.yaml

```diff
@@ -162,22 +162,17 @@ properties:
     type: "string"
   BRANDING_FILEPATHS:
     default:
-      css: null
       favicon: null
-      header_bullet: null
       icon_16: null
       icon_32: null
       icon_180: null
       icon_192: null
       icon_mask: null
-      javascript: null
       logo: null
-      nav_bullet: null
```

Comment:
Probably need to update the actual default in `settings.py` correspondingly to get tests passing.